Lib9c/Currencies.cs

```diff
@@ -36,6 +37,41 @@ public static class Currencies
             0,
             minters: null);
 
+        public static readonly Currency Garage = Currency.Uncapped(
+            "Garage",
```

Comment:
AFAIK, we have `NCG` and `CRYSTAL` for now.
Is it ok to name `Garage` not `GARAGE`?
I think all capital ticker is good to recognize and can reduce mistakes.

Reply:
Oh I didn't think so. I'll capitalize it for consistency. Thanks!